builder/amazon/common/access_config.go

```diff
@@ -83,8 +86,10 @@ func (c *AccessConfig) Region() (string, error) {
 func (c *AccessConfig) Prepare(ctx *interpolate.Context) []error {
 	var errs []error
 	if c.RawRegion != "" {
-		if valid := ValidateRegion(c.RawRegion); valid == false {
-			errs = append(errs, fmt.Errorf("Unknown region: %s", c.RawRegion))
+		if c.SkipValidation == false {
```

Comment:
As above, `if !c.SkipValidation` and this series of nested conditionals can be collapsed in a similar way.